runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -1168,6 +1169,53 @@ J9::Z::TreeEvaluator::inlineVectorizedStringIndexOf(TR::Node* node, TR::CodeGene
    TR::Register* patternLenReg = cg->gprClobberEvaluate(node->getChild(firstCallArgIdx+3));
    TR::Register* stringIndexReg = cg->gprClobberEvaluate(node->getChild(firstCallArgIdx+4));
 
+   TR::Register* stringValueDataStart = NULL;
+   TR::Register* patternValueDataStart = NULL;
+
+   // Offset to be added to array object pointer to get to the data elements
+   int32_t offsetToDataElements = static_cast<int32_t>(TR::Compiler->om.contiguousArrayHeaderSizeInBytes());
+#ifdef J9VM_GC_SPARSE_HEAP_ALLOCATION
+   if (TR::Compiler->om.isOffHeapAllocationEnabled())
+      {
+      srm = cg->generateScratchRegisterManager(11);
```

Comment:
Why can't we simply clobberEvaluate `node->getChild(firstCallArgIdx)` and other one ?